.github/workflows/system-tests-centos7.yml

```diff
@@ -0,0 +1,39 @@
+name: ParallelCluster Cookbook System Test CentOS 7
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
probably can be removed or the branch needs to be fixed since we don't use master